packages/blade/src/components/Typography/Text/Text.tsx

```diff
@@ -55,7 +56,8 @@ const getTextProps = <T extends { variant: TextVariant }>({
   weight,
   size,
   contrast,
-}: Pick<TextProps<T>, 'type' | 'variant' | 'weight' | 'size' | 'contrast'>): Omit<
+  textAlign,
+}: Pick<TextProps<T>, 'type' | 'variant' | 'weight' | 'size' | 'contrast' | 'textAlign'>): Omit<
```

Comment:
textAlign added?

Reply:
Good catch, was for a different PR. Removed